datafusion/core/src/physical_plan/windows/window_agg_exec.rs

```diff
@@ -122,7 +122,9 @@ impl ExecutionPlan for WindowAggExec {
     }
 
     fn output_ordering(&self) -> Option<&[PhysicalSortExpr]> {
-        self.input.output_ordering()
+        // This executor maintains input order, and has required input_ordering filled
+        // hence output_ordering would be `required_input_ordering`
+        self.required_input_ordering()[0]
```

Comment:
In order to be a little easier to read, we can use `self.sort_keys.as_deref()` here. This would avoid the construction of a single-element vector and then taking the first element of it. The downside of doing this would be replicating this line from `required_input_ordering`, but I think the added clarity is worth it here. Thoughts, @alamb?

Reply:
that would also be good